Loop/Managers/TrustedTimeChecker.swift

```diff
@@ -36,6 +36,10 @@ class TrustedTimeChecker {
 
     init(_ alertManager: AlertManager) {
         ntpClient = TrueTimeClient.sharedInstance
+        ntpClient.logCallback = { [weak self] _ in
+            // Too chatty
+//            self?.log.debug("%@", $0)
```

Comment:
[nit] Remove commented out code?

Reply:
I left it in there in case someone may want to see the logging.  I'll make the comment clearer...